internal/dump/common.go

```diff
@@ -74,13 +75,20 @@ func dumpTree(ctx context.Context, repo restic.Repository, rootNode *restic.Node
 // GetNodeData will write the contents of the node to the given output.
 func GetNodeData(ctx context.Context, output io.Writer, repo restic.Repository, node *restic.Node) error {
 	var (
-		buf []byte
-		err error
+		buf       []byte
+		ok        bool
+		err       error
```

Comment:
This needs a gofmt, I think.

Reply:
D'oh. It's actually because of gofmt, but then I updated the code with @MichaelEischer's suggestion above, and did it all wrong. gofmt'd again...